kafka-connect-jms/src/main/scala/com/datamountaineer/streamreactor/connect/jms/sink/converters/ProtoStoredAsConverter.scala

```diff
@@ -0,0 +1,169 @@
+package com.datamountaineer.streamreactor.connect.jms.sink.converters
+
+import com.datamountaineer.streamreactor.common.converters.ParserImpl
+import com.datamountaineer.streamreactor.connect.jms.config.JMSSetting
+import com.github.os72.protocjar.Protoc
+import com.google.protobuf.util.JsonFormat
+import com.google.protobuf.{DescriptorProtos, Descriptors, DynamicMessage, TypeRegistry}
+import com.typesafe.scalalogging.StrictLogging
+import org.apache.kafka.connect.errors.DataException
+import org.apache.kafka.connect.json.{JsonConverter, JsonConverterConfig}
+import org.apache.kafka.connect.sink.SinkRecord
+
+import java.io.{File, FileInputStream, IOException}
+import java.nio.charset.StandardCharsets
+import java.nio.file.attribute.BasicFileAttributes
+import java.nio.file.{Files, Path, Paths}
+import java.util
+import java.util.Collections
+import java.util.concurrent.ConcurrentHashMap
+import java.util.stream.Collectors
+import scala.collection.JavaConverters._
+import scala.util.{Failure, Success, Try}
+
+case class ProtoStoredAsConverter() extends ProtoConverter with StrictLogging {
+
+  private val SCHEMA_PROTO_PATH = "proto_path"
+  private val SCHEMA_PROTO_FILE = "proto_file"
```

Comment:
should probably be dot separated as they are settings used in connector config  for the val, e.g. proto.path and proto.file

Reply:
using dot separation in STOREDAS has some lexical parser issues so used underscrore instead!